lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -571,6 +529,23 @@ def write(chunk)
             determine_entry_level_monitored_resource_and_labels(
               group_level_resource, group_level_common_labels, record)
 
+          unless @logs_viewer_address_displayed
+            instance_resource_types = [
```

Comment:
This should be a constant in `ServiceConstants`. Also, this should probably be a Set.